JVMConsole/build.gradle.kts

```diff
@@ -2,18 +2,29 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
     kotlin("jvm") version "1.6.10"
-    id("io.realm.kotlin") version "0.9.0"
     application
 }
+
+buildscript {
+    dependencies {
+        classpath("io.realm.kotlin:gradle-plugin:0.10.0")
+    }
+}
+rootProject.extra["realmVersion"] = "0.10.0"
+
+apply(plugin = "io.realm.kotlin")
+
 group = "io.realm.example"
-version = "1.0"
+version = "0.10.0"
```

Comment:
don't forget to update its Readme to use 
```
java -jar build/libs/JVM_Console-0.10.0.jar
```

Reply:
Good catch!